src/auth-service/controllers/create-user.js

```diff
@@ -286,6 +286,58 @@ const createUser = {
       });
     }
   },
+  deleteMobileUserData: async (req, res) => {
+    try {
+      let responseFromDeleteAppData;
+      logText("We are deleting the app data.....");
+      const { uid } = req.query;
+      const { ct } = req.query;
+
+      let request = {};
+      request["body"] = {};
+      request["body"]["uid"] = uid;
+      request["body"]["ct"] = ct;
+      logObject("request:", request);
+     try {
+       responseFromDeleteAppData = await createUserUtil.deleteMobileUserData(
+         request
+       );
+     } catch (error) {
+       console.error(error);
```

Comment:
We have an opportunity to also log this via Slack.
`logger.error(`internal server error -- ${error.message}`);`
Also, we need to take advantage of the global logger, that is: ` logObject("error", error)`